client/src/app/app.routes.ts

```diff
@@ -41,46 +43,49 @@ export const rootRouterConfig: Routes = [
         path: 'my-budget', canActivate: [AuthenticationGuard],
         children: [
             { path: '', redirectTo: currentYear, pathMatch: 'full' },
-            { 
+            {
                 path: ':year', component: MyBudgetComponent, canActivate: [AuthenticationGuard],
                 children: [
                     { path: 'create-request/:budgetId', component: NewRequestModalComponent, canActivate: [AuthenticationGuard] },
+                    { path: 'create-team-request/:year', component: NewTeamRequestModalComponent, canActivate: [AuthenticationGuard] },
                     { path: 'request/:requestId', component: RequestDetailModalComponent, canActivate: [AuthenticationGuard] },
-                    { path: 'request/:requestId/edit', component: EditRequestModalComponent, canActivate: [AuthenticationGuard] }
+                    { path: 'request/:requestId/edit', component: EditRequestModalComponent, canActivate: [AuthenticationGuard] },
+                    { path: 'request/:requestId/edit/:type', component: EditRequestModalComponent, canActivate: [AuthenticationGuard] },
                 ]
             }
         ]
     },
     { path: 'create-request', component: RequestAddComponent, canActivate: [AuthenticationGuard] },
+    { path: 'create-team-request', component: TeamRequestAddComponent, canActivate: [AuthenticationGuard] },
     {
         path: 'requests', component: RequestsComponent, canActivate: [ViewerGuard],
         children: [
             { path: '', redirectTo: 'pending/' + currentYear, pathMatch: 'full' },
             {
                 path: 'pending/:year', component: RequestListComponent, data: { filter: RequestFilter.Pending }, canActivate: [ViewerGuard],
                 children: [
-                    { path: 'detail/:requestId', component: RequestDetailModalComponent, canActivate: [ViewerGuard] },
+                    { path: 'detail/:requestId/type/:type', component: RequestDetailModalComponent, canActivate: [ViewerGuard] },
```

Comment:
If you need to distinguish between request types (which IMHO maked sense, if we want to be able to add/remvoe employees from the team requests), it is probable better to have separate routes